2016/proposals/chris-medrela.md

```diff
@@ -0,0 +1,461 @@
+- **Trainees Spreadsheet**: Google Docs Spreadsheet with list of trainees. The columns are:
+    - name
+    - contact email
+    - which training course they took part in
+    - the name of the Software Carpentry lesson they're concentrating on (if any)
+    - the name of the Data Carpentry lesson they're concentrating on (if any)
+    - the URL of their Software Carpentry exercise submission (if any)
+    - whether or not they have submitted something for Data Carpentry
+    - the date and moderator of the discussion session they have taken part in (we currently store both values in one column)
+    - the date on which they passed their Software Carpentry lesson demo (if any)(this date becomes the award date of the PDF certificate)
+    - the date on which they passed their Data Carpentry lesson demo (if any)
+- **Partners Spreadsheet**: Google Docs Spreadsheet with negotiations with partners about instructor training sessions.
+- **Individual Application Form** -- a form in Amy for Candidates to apply as an individual. Temporarily this form is disabled.
+- **Individual Application Spreadsheet** -- a Google spreadsheet for individual applications. The columns are:
+    - name
+    - contact address
+    - location
+    - why they want to become an instructor
+- **Group Application Form** and **Group Application Spreadsheet**: Google web form and a spreadsheet for group applications (should be moved to Amy as workshop-style events). The Group Application Spreadsheet contains:
+    - all names and email addresses (principal contact highlighted)
+    - group location
+    - description of what the group plan to do once they're trained
+- **Calendar**: Software Carpentry's community calendar (a Google calendar).
+- **New Instructor Form**: [Form on AMY](https://amy.software-carpentry.org/workshops/update_profile/) for people who passed exam.
+- **Database**, **Amy Database**: Amy database with list of certified instructors.
+- **SWC Lesson Changes**: GitHub pull requests to SWC Lessons. First step of Checkout Process.
+- **DC Lesson Changes Doc**: Google Docs page for Data Carpentry. First step of Checkout Process.
+- **SWC/DC Lessons**: SWC/DC lessons available on SWC website.
+
+### Other
+
+- **Instructor Training**, **Course**
+- **Checkout Process**, **Checkout**: process of becoming Certified Instructor that takes place after Instructor Training and, at this moment, consists of three steps: 1) Homework 2) Discussion Session 3) Checkout Session
+- **Homework**: Lesson Changes; first step of Checkout Process.
+- **Pre-Checkout discussion session**, **Discussion Session**, **Discussion**: meeting between an Instructor Trainer and some trainees to discuss one of Software or Data Carpentry lessons.
+- **Checkout Session**, **Final Demonstration Session**: meeting between a Instructor Trainer and a trainee to conclude Instructor Training.
+- **Carpentry** -- either SWC or DC.
+
+## The Current Workflow
```

Comment:
Where should I put those sections? Note that they're not the same like [your pull request](https://github.com/swcarpentry/board/pull/81/files) since I've modified it a bit, to summarize discussion that took place in the pull request.